incubator/fireflyiii/1.0.0/ix_values.yaml

```diff
@@ -0,0 +1,41 @@
+##
+# This file contains Values.yaml content that gets added to the output of questions.yaml
+# It's ONLY meant for content that the user is NOT expected to change.
+# Example: Everything under "image" is not included in questions.yaml but is included here.
+##
+
+image:
+  repository: jc5x/firefly-iii
+  pullPolicy: IfNotPresent
+  tag: latest
+
+env:
+  DB_USERNAME: firefly
+  DB_DATABASE: firefly
+  DB_CONNECTION: pgsql
+  DB_PORT: 5546
+
+envValueFrom:
+  DB_HOST:
+    secretKeyRef:
+      name: dbcreds
+      key: postgresql_host
+  DB_PASSWORD:
+    secretKeyRef:
+      name: dbcreds
+      key: postgresql-password
+
+# Enabled postgres
+# ... for more options see https://github.com/bitnami/charts/tree/master/bitnami/postgresql
+postgresql:
+  enabled: true
+  postgresqlUsername: fireflyiii
+  postgresqlDatabase: fireflyiii
+  existingSecret: dbcreds
+  persistence:
+    enabled: true
+    existingClaim: db
```

Comment:
Make sure to copy the "ersistence" section from Bitwarden as well, including the DB section there...
That creates the "existing claim" referenced here ;)